specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/EventHub-preview.json

```diff
@@ -70,6 +70,39 @@
         }
       }
     },
+    "/providers/Microsoft.EventHub/availableClusters": {
```

Comment:
POST actions should have a verb as the first portion of the action name, i.e. `listAvailableClusters`.  However, what is the reason to make this a POST rather than a GET? Similar APIs (/skus, /locations, /providers, etc) are commonly modeled as GET methods

Reply:
No good reason. I have modified both namespace enumeration in cluster and the available clusters enumeration from POSTs to GETs.

Reply:
@FBettati  - Thanks, but this should return a collection response:
``
"value": [
{...},
{...}
]``

Reply:
@KrisBash - For requests with this URI, we're only attempting to provide a mapping from region to count of available clusters. This API's primary purpose is for validation checks in Portal and other clients.

Is your recommendation to have a response of this structure: `"value": [ { "westus": 5 }, { "eastus": 4 } ]` ? 

And, expanding on that, if we add a collection response, then by ARM guidelines wouldn't we also have to add a individual scope to each item in the collection as well (`/providers/Microsoft.EventHub/availableCluster/{someName}`)?